src/pycram/ros_utils/robot_state_updater.py

```diff
@@ -0,0 +1,88 @@
+import atexit
+from datetime import timedelta
+
+import tf
+import time 
+
+from geometry_msgs.msg import TransformStamped
+from sensor_msgs.msg import JointState
+from typing_extensions import Optional
+
+from ..datastructures.world import World
+from ..robot_description import RobotDescription
+from ..datastructures.pose import Pose
+from ..ros.data_types import Time, Duration
+from ..ros.ros_tools import wait_for_message, create_timer
+
+
+class WorldStateUpdater:
```

Comment:
Why is the name changed if this is still just the robot_state_updater?

Reply:
because it also updates the locations of the objects in the world